src/crypto/verification/request/VerificationRequest.ts

```diff
@@ -458,6 +459,13 @@ export class VerificationRequest<C extends IVerificationChannel = IVerificationC
         return this._verifier!;
     }
 
+    public async startVerification(method: string): Promise<Verifier> {
```

Comment:
Is it desirable to have a list of supported methods here, rather than using `string` as a type?

Reply:
possibly, though we do have to support `string`s on all the other methods for backwards compatibility, so it's not an obvious win.